### PR TITLE
[ReadyToMerge] KCD Amsterdam 2019

### PR DESCRIPTION
**Pull request by ams0**
For historical reasons we want to keep the 2019 edition of Kubernetes Community Day Amsterdam here with the actual list of sponsors, talks and sessions.

### CODE REVIEW

**Comment by iennae** [comment]
Blocked by https://github.com/cncf/kubernetes-community-days/issues/250

**Comment by iennae** [comment]
Also https://github.com/cncf/kubernetes-community-days/issues/249 should get resolved before this is merged (unless folks approve merging)

**Comment by lucperkins** [comment]
@iennae @ams0 The logo sizing issue has been addressed. @ams0, could you rebase against the latest changes?

**Comment by ams0** [comment]
@lucperkins logo sizing looks great. Merging #258 and rebasing will make this ready to merge too. Thanks!

**Comment by dankohn** [comment]
#258 is merged. Please merge master into this branch (or I can do for you if want) and then add an LGTM when this is ready to go.

**Comment by ams0** [comment]
Looking good! But we still have no program because we have no paid sched account. I have the sessions and speakers in .xls format ready to be imported.

**Comment by ams0** [comment]
Yes, of course, the program won't be needed until after the CFP closes (15 May). Then please merge away!

File: content/events/2019-amsterdam/_index.md [hunk 1]
```diff
@@ -1,12 +1,195 @@
...
+    logo: sponsors/lunatech.png
+    url: https://www.lunatech.com/
+  - name: CloudBees
+    logo: sponsors/cloudbees.png
+    url: https://www.cloudbees.com/
+  - name: Vamp
+    logo: sponsors/vamp.jpeg
+    url: https://www.vamp.io/
+  - name: Netapp
+    logo: sponsors/netapp.jpg
```
Comment (dankohn) [inline content/events/2019-amsterdam/_index.md]: Please use an SVG for all images. They're all at landscape.cncf.io

Comment (ams0) [reply content/events/2019-amsterdam/_index.md]: Addressed for most sponsors (a few are not CNCF members). Should look better.

**Pull request merged**